features/admins/admin_works.feature

```diff
@@ -6,7 +6,7 @@ Feature: Admin Actions for Works, Comments, Series, Bookmarks
   Scenario: Can troubleshoot works
     Given I am logged in as "regular_user"
       And I post the work "Just a work you know"
-    When I am logged in as superadmin
+    When I am logged in as a "policy_and_abuse" admin
```

Comment:
fwiw, Support or Tag Wrangling usually does this one.

Reply:
I'll give this scenario to Support.